docs/advanced/monitoring/applications-insights.md

```diff
@@ -32,18 +37,30 @@ namespace Example
 
     public class Program
```

Comment:
Is it maybe worth focusing the sample on the new host builders? Not perfect i suspect but see here: https://github.com/RDalziel/MTDiagnostic/blob/master/MTDiagnosticLogging/Program.cs

Reply:
Totally, in fact, creating a sample (Sample-ApplicationInsights) of an API that uses MassTransit similar to my whole unit test would be awesome thanks. Happy to create  repo on the org for you to push it :)

Reply:
Be happy to ! :)

Reply:
this is fantastic, glad to see all the hard work on this is in.